pkg/registry/core/service/strategy.go

```diff
@@ -205,6 +210,17 @@ func topologyKeysInUse(svc *api.Service) bool {
 	return len(svc.Spec.TopologyKeys) > 0
 }
 
+// returns true if svc.Spec.ServiceIPFamily field is in use
```

Comment:
`svc.Spec.AllocateLoadBalancerNodePort`

Reply:
I think this comment was missed. Should be `AllocateLoadBalancerNodePort` and not `ServiceIPFamily` here.